Makefile

```diff
@@ -4,6 +4,11 @@
 #
 # Run make (with no arguments) to see help on what targets are available
 
+# enable parallel builds by default
+# it can be overridden from make command line using -jN
+NRCORES:=$(nproc)
```

Comment:
For me `NCORES` looks like a more intuitive name than `NRCORES`... just a suggestion...